alonzo/impl/src/Cardano/Ledger/Alonzo/TxWitness.hs

```diff
@@ -178,29 +214,6 @@ pattern TxWitness {txwitsVKey, txwitsBoot, txscripts, txdats, txrdmrs} <-
 
 {-# COMPLETE TxWitness #-}
 
--- ======================================================================
-
--- | Right-biased semigroup - if there are (somehow) multiple entries either for
--- a given 'ScriptHash' or a given 'Data', this will bias to the entry on the
--- right. Note that this should not happen in practise.
-instance
-  (Era era, ToCBOR (Core.Script era)) =>
-  Semigroup (TxWitness era)
-  where
-  TxWitness a b c d e <> TxWitness a' b' c' d' e' =
-    TxWitness
-      (a `Set.union` a')
-      (b `Set.union` b')
-      (c <> c')
-      (d <> d')
-      (e <> e')
```

Comment:
I removed the `Semigroup` instance here since `e` is a `MemoBytes` and I was worried that this could lead someone to remove the stored serialization accidentally. I can add it back if there's really no worry...